docs/user/user/adding-glean-to-your-project/javascript.md

```diff
@@ -0,0 +1,140 @@
+# Adding Glean to your Javascript project
+
+Javascript projects use the Javascript APIs provided by Glean.js.
+
+Currently, these bindings support collecting data from [Browser Extensions](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions) (cross-browser) and [Qt/QML applications](https://doc.qt.io/qt-5/qtqml-index.html).
+
```

Comment:
I am wondering if we should circle back to the integration checklist on the first paragraph of these pages. 

Each page will go in detail about how to "Integrate Glean into the buiild system" for each language binding, but there are more steps to integrating Glean fully that are listed in the integration checklist.

Something like:

> This page provides a step-by-step guide on how to integrate the Glean library into a Javascript  project. Nevertheless this is just one of the required steps for integrating Glean succesfully into a project.  Check you the full Glean integration checklist for a comprehensive list of all the steps involved in doing so.

Reply:
That's a great idea.